### DISCUSSION TITLE
Support Alfa adapters

### DISCUSSION THREAD

**Discussion by ruslan-gennadievich** [svpcom/wfb-ng#353] [Questions]
Hi! For example: Alfa AWUS036NHR, ALFA AWUS036H
As I know this 802.11 with support injection. This adapters recommended by Kali Linux and other pentest distributive. Also I see you author of patched RTL8812\14 driver. This driver is required for correct work wfb-ng? Or I can use chipset like Atheros AR9271L ?

Thanks!

**Comment by svpcom** [comment]
wfb-ng support any wifi adapter with injection support in drivers. But different drivers have different quality of injections and/or packet capture. I recommend 8812au or 8812eu adapters because I've tested (and patched) their drivers myself. But you can use any third-party drivers if they will work for you

**Comment by ruslan-gennadievich** [comment]
Thankyou for reply. I will test this adapters and write feedback
